connectors/kinesis-data-streams-connector/src/main/scala/zio/connect/kinesisDataStreams/KinesisDataStreamsConnector.scala

```diff
@@ -0,0 +1,27 @@
+package zio.connect.kinesisDataStreams
```

Comment:
I would drop the camelcase format. We should have `zio.connect.kinesisdatastreams` instead.